bus-mapping/src/circuit_input_builder/execution.rs

```diff
@@ -206,14 +204,15 @@ impl ExecState {
 }
 
 /// Defines the various source/destination types for a copy event.
-#[derive(Clone, Copy, Debug, PartialEq, Eq)]
+#[derive(Clone, Copy, Debug, PartialEq, Eq, EnumIter, Default)]
 pub enum CopyDataType {
     /// When we need to pad the Copy rows of the circuit up to a certain maximum
     /// with rows that are not "useful".
     Padding,
     /// When the source for the copy event is the bytecode table.
     Bytecode,
     /// When the source/destination for the copy event is memory.
+    #[default]
```

Comment:
want to know why `Memory` needs additional `#[default]` mark?

Reply:
the derive macro needs to know which variant is the default. i picked `Memory` because that was the existing default value.